### ISSUE TITLE
Change width-height of PasswordGesture component

### ISSUE THREAD

**Issue by angelos3lex** [Spikef/react-native-gesture-password#31]
Hi, even if we can override the styles of the component, we can't override height or width as they are fixed const. And they are used everywhere in file to set layout's height/width.
For example, by:
```
 <PasswordGesture
          style={{width:400, height:680}}
          ref="pg"
          status={this.state.status}
          message={this.state.message}
          onStart={() => this.onStart()}
          onEnd={password => this.onEnd(password)}
        />
```
Absolutely nothing happens.
Is there any option to alter it's height/width? 
In fact, i want to enter the password-gesture inside a popup window, but i need to override `Width = Dimensions.get('window').width` and `Height = Dimensions.get('window').height` with parent's height/width, which is the size of the window popup .
So to sum up, my question is:
Is there any way to make the GesturePassword inherit height/width from parent view? 
This issue was also referenced in [#16](https://github.com/Spikef/react-native-gesture-password/issues/16) but closed without fix.

**Comment by JZorina** [comment]
did you managed to deal with it?

**Comment by angelos3lex** [comment]
@JZorina Nope.. I've forked another similar repo [here](https://github.com/angelos3lex/react-native-smart-gesture-password-angeloslex)
And if you can take a look at the code in usage example in readme, at the last line i have a comment about how to place it inside a dialog(or another parent view). That fork works for me on production without any problem + it's not outdated.  Maybe a similar workaround can work for this one as well
